utensor_cgen/transformer/linear_reoder.py

```diff
@@ -79,4 +80,57 @@ def transform(self, ugraph):
       topologic_order_graph(ugraph)
       graph_validate(ugraph)
 
-    return ugraph
\ No newline at end of file
+    return ugraph
+
+
+class LinearReorderTransformerV2(Transformer):
+  METHOD_NAME = 'linear_reorder_v2'
+  KWARGS_NAMESCOPE = '_linear_reorder_v2'
+
+  def __init__(self):
+    self.prune_graph = False
+
+  @property
+  def pattern_ugraph(self):
+    graph = tf.Graph()
```

Comment:
Probably an insanely stupid question, but what does this subgraph have to do with LinearReorder?

Reply:
You mean the naming?
I just follow @neil-tan's naming.

Reply:
@neil-tan The hell are you doing here? XD

Reply:
I know this.
Just rearrange the layers to save memory.
`relu -> maxpool` is equivalent to `maxpool -> relu`
And @neil-tan call this linear_reorder.

Reply:
@mbartling this is to reduce the memory needed to store the activation by taking advantage of down-sampling